modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/metadata/OperationMetadataTestCase.java

```diff
@@ -287,6 +287,38 @@ public void dynamicOutputAndContentWithCache() throws Exception
     }
 
     @Test
+    public void shouldInheritOperationResolvers() throws Exception
+    {
+        processorId = new ProcessorId(SHOULD_INHERIT_OPERATION_RESOLVERS, FIRST_PROCESSOR_INDEX);
+
+        final OperationMetadataDescriptor metadataDescriptor = getOperationDynamicMetadata();
+
+        assertExpectedOutput(metadataDescriptor.getOutputMetadata(), personType, void.class);
+
+        assertThat(metadataDescriptor.getParametersMetadata().size(), is(1));
+        assertExpectedType(metadataDescriptor.getParametersMetadata().get(0), "type", String.class);
+
+        assertThat(metadataDescriptor.getContentMetadata().isPresent(), is(true));
+        assertExpectedType(metadataDescriptor.getContentMetadata().get(), "content", personType);
+    }
+
+    @Test
+    public void shouldInheritExtensionResolvers() throws Exception
+    {
+        processorId = new ProcessorId(SHOULD_INHERIT_EXTENSION_RESOLVERS, FIRST_PROCESSOR_INDEX);
+
+        final OperationMetadataDescriptor metadataDescriptor = getOperationDynamicMetadata();
```

Comment:
this assertion is exactly the same as in the prior test... extract to reusable method.